components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/DateRangePicker.jsx

```diff
@@ -175,13 +177,19 @@ export default class DateRangePicker extends Widget {
         if (availableGranularities.indexOf(this.capitalizeCaseFirstChar(granularity)) > -1) {
             return granularity;
         } else {
-            return this.lowerCaseFirstChar(availableGranularities[0]);
+            return availableGranularities[0].toLowerCase;
         }
     }
 
     getDefaultTimeRange() {
-        let defaultTimeRange = this.state.options.defaultValue;
-        let minGranularity = this.state.options.availableGranularities;
+        let defaultTimeRange = '3 Months';
+        if (this.state.options.defaultValue) {
```

Comment:
Let's write this in a single line
`const defaultTimeRange = this.state.options.defaultValue || '3 Months'`
Do same in similar other places

Reply:
Fixed and merged in PR #63